contrib/99designs/gqlgen/example_test.go

```diff
@@ -0,0 +1,40 @@
+// Unless explicitly stated otherwise all files in this repository are licensed
+// under the Apache License Version 2.0.
+// This product includes software developed at Datadog (https://www.datadoghq.com/).
+// Copyright 2016-2019 Datadog, Inc.
+
+package gqlgen_test
+
+import (
+	"log"
+	"net/http"
+	"os"
+
+	"github.com/99designs/gqlgen/handler"
+
+	"gopkg.in/DataDog/dd-trace-go.v1/contrib/99designs/gqlgen"
+	"gopkg.in/DataDog/dd-trace-go.v1/contrib/99designs/gqlgen/example"
+)
+
+const defaultPort = "8080"
+
+func ExampleNew() {
+	port := os.Getenv("PORT")
```

Comment:
This doesn't add anything to the example. Let's just use whatever port works.

Reply:
👍